docs/api/1.x/_details-patch-object.rst

```diff
@@ -82,8 +65,20 @@ For more information about each operation, please refer to
 `JSON-Patch Specification <https://tools.ietf.org/html/rfc6902>`_.
 
 This is very useful when altering permissions since there is no need to get
-the current value before adding some principal to the list. Value is not used
-on permission operations and can be omitted.
+the current value before adding some principal to the list.
+
+Permissions
+-----------
+
+.. In the JSON request payloads, at least one of ``data`` and ``permissions``
+.. must be provided.
+
+When permissions are modified, the same behaviour used for the body is expected.
+But notice thought that there are some permissions specific behaviours, such as:
```

Comment:
I think `but` and `though` (N.B. typo) are redundant -- either one is fine. I'd probably write `note` instead of `notice`. `permissions-specific`.